src/main/webapp/app/shared/profile-picture/profile-picture.component.ts

```diff
@@ -0,0 +1,50 @@
+import { Component, OnChanges, OnInit, input } from '@angular/core';
+import { faCog } from '@fortawesome/free-solid-svg-icons';
+import { ArtemisSharedCommonModule } from 'app/shared/shared-common.module';
+import { ArtemisSharedPipesModule } from 'app/shared/pipes/shared-pipes.module';
+import { getInitialsFromString } from 'app/utils/text.utils';
+import { getBackgroundColorHue } from 'app/utils/color.utils';
+import { RouterLink } from '@angular/router';
+
+@Component({
+    selector: 'jhi-profile-picture',
+    standalone: true,
+    templateUrl: './profile-picture.component.html',
+    styleUrls: ['./profile-picture.component.scss'],
+    imports: [ArtemisSharedCommonModule, RouterLink, ArtemisSharedPipesModule],
+})
+export class ProfilePictureComponent implements OnInit, OnChanges {
+    readonly imageSizeInRem = input<string>('2.15');
+    readonly fontSizeInRem = input<string>('0.9');
```

Comment:
As far as I can tell they always have the same ratio for all inputs. Can we only have one as input and calculate the other?

Reply:
I was trying to do that but unfortunately the font size is not in a direct ratio with the imagesize for most things. Tutorial Group is 4.5:2 = 2.25, here it is 2.15:0.9 = 2.38, in the communication channel lists its 2:0.8=2.5, and so on.